src/hotspot/share/gc/shared/blockOffsetTable.hpp

```diff
@@ -49,12 +49,13 @@
 class ContiguousSpace;
 
 class BOTConstants : public AllStatic {
-public:
-  static uint LogN;
-  static uint LogN_words;
-  static uint N_bytes;
-  static uint N_words;
+private:
+  static uint _LogN;
+  static uint _LogN_words;
+  static uint _N_bytes;
+  static uint _N_words;
```

Comment:
The `private` visibility modifier can be removed as this is default at the top of a class.
The static variables should start with a lower case letter after the underscore, something like `_log_n`.

My suggestion would also be to change `N`/`n` to something more understandable, like `size`, and add `block`, i.e. something like `_log_block_size`, `_log_block_size_in_words` similar to the corresponding `CardTable` members etc.

Reply:
As I can understand, we need to replace "N" with something meaningful. Does something like "entry_size" or "bot_entry_size" would work?